src/lib/seam/use-seam-client.ts

```diff
@@ -57,9 +62,14 @@ export function useSeamClient(): {
       })
 
       if (!res.ok || res.client_session?.token == null) {
-        throw new Error('Failed to get client access token')
+        throw new Error('Failed to get client session token')
       }
 
+      telemtry.identify(userIdentifierKey, {
```

Comment:
@seveibar This is where we can identify the "user". We need to choose a unique id. Maybe it should be the workspace id?

Reply:
`${workspaceId}_${userIdentifierKey}` makes the most sense to me

Reply:
We need to update the API to return the workspace id in the client session response. For cases where we only have the token, we need to make a request to get the client session.